src/views/LoansListing/LoansListing.js

```diff
@@ -29,7 +29,16 @@ class LoansListing extends React.Component {
     match: PropTypes.object,
   };
 
-  /**
+    // eslint-disable-next-line react/sort-comp
```

Comment:
Don't disable lint. The reformatting here is minor; please just do what it asks :) You can rearrange things here, or move the `createRef` line into a constructor. The acctual cDM methods looks good, of course.